### PR TITLE
[Form] [Security] Add missing pt_BR translations

### LINKED ISSUE #38729: Missing translations for Brazilian Portuguese (pt_BR)
Hello,

There are some missing translation for Brazilian Portuguese (pt_BR). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.pt_BR.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.pt_BR.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by adielcristo**
| Q             | A
| ------------- | ---
| Branch?     | 3.4
| Bug fix?     | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38729
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf [hunk 1]
```diff
@@ -64,7 +64,15 @@
             </trans-unit>
             <trans-unit id="16">
                 <source>Account is locked.</source>
-                <target>A conta está travada.</target>
+                <target>A conta está bloqueada.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf]: I think using "travada" is better here. 

It should be "locked", and not "blocked". 
Ie, you can "lock" an issue on Github without "blocking" anyone.

**Review by Nyholm — COMMENTED** [review]
Thank you very much. I just have one small comment

File: src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf (hunk 1 above)
Comment (adielcristo) [reply src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf]: Thanks for the suggestion. Fixed.

**Review by Nyholm — APPROVED** [review]
Thank you. 

Im happy with this PR 👍

**Comment by derrabus** [comment]
Thank you Adiel.

**Pull request merged**